dex/src/main/scala/com/wavesplatform/dex/actors/address/AddressActor.scala

```diff
@@ -64,6 +64,7 @@ class AddressActor(
   recovered: Boolean,
   blockchain: BlockchainInteraction,
   settings: AddressActor.Settings = AddressActor.Settings.default,
+  lpAccounts: Set[PublicKey] = Set.empty[PublicKey],
```

Comment:
we can pass here sth like ```isLpAccount: Address => Boolean``` . Internally it should have memoization in order to avoid doing ```lpAccounts.map(_.toAddress)``` on each AddressActor